ProjFS.Mac/PrjFS.xcodeproj/project.pbxproj

```diff
@@ -934,6 +996,60 @@
 			};
 			name = "Profiling(Release)";
 		};
+		F5E39C7C21F1118D006D65C2 /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				CODE_SIGN_STYLE = Automatic;
+				COMBINE_HIDPI_IMAGES = YES;
+				DEVELOPMENT_TEAM = UBF8T346G9;
+				INFOPLIST_FILE = PrjFSKextTests/Info.plist;
+				LD_RUNPATH_SEARCH_PATHS = (
+					"$(inherited)",
+					"@executable_path/../Frameworks",
+					"@loader_path/../Frameworks",
+				);
+				MACOSX_DEPLOYMENT_TARGET = 10.14;
```

Comment:
While playing around with this code, I noticed that the deployment target is explicitly set to 10.14 in the test suite target. That should probably just be inherited from the project, which uses 10.13.